web/src/i18n/locales/en.yaml

```diff
@@ -27,18 +28,8 @@
       yearsCovered: "What year(s) does this APD cover?"
       labels: 
         desc: "Program Introduction"
-    keyPersonnel: 
-      title: "Key Personnel"
-      helpText: "This section includes any personnel associated with this APD, and includes their cost amount and percentage full-time equivalent (FTE)."
-      labels: 
-        entryNum: "#"
-        title: "Title"
-        desc: "Description"
-        yearCost: "{{year}} Cost"
-        costAmt: "Amount"
-        costPerc: "% FTE"
-      removeLabel: "Remove personnel"
-      addButtonText: "Add personnel"
+    previousActivitySummary:
```

Comment:
Should this be under `previousActivities.outline`?  Since it's displayed in the "previous activities / outline" subsection.

Reply:
yes, good call